setup.py

```diff
@@ -153,6 +166,7 @@ def write_version(filename=os.path.join(*['airflow',
 ]
 # major update coming soon, clamp to 0.x
 cloudant = ['cloudant>=0.5.9,<2.0']
+codecov = ['codecov']
```

Comment:
Why not just put codecov in devel_ci/devel_all list. I don't think we need an extra group just for codecov as it's not something that a user would want at runtime.

Reply:
Ok. I will simplify it. No problem.

Reply:
Done. Moved to devel. Makes sense to keep it there.